Src/Base/AMReX_RealVect.H

```diff
@@ -672,6 +714,36 @@ RealVect::scale (Real s) noexcept
   return *this;
 }
 
+AMREX_GPU_HOST_DEVICE
+inline
+IntVect
+RealVect::floor () const noexcept
+{
```

Comment:
are you sure these functions ('floor', 'ceil', and 'round') have device implementations in the versions of CUDA we care about (say, 9.2 and later)?

Reply:
I found this in the documentation
https://docs.nvidia.com/cuda/archive/9.2/cuda-math-api/group__CUDA__MATH__DOUBLE.html